templates/taxi/car_list.html

```diff
@@ -7,6 +8,10 @@ <h1>
       Create
     </a>
   </h1>
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input class="btn btn-secondary" type="submit" value="🔎">
+  </form>
```

Comment:
Create a separate template for search form and include it where you need to avoid code duplication